.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -509,7 +509,8 @@ def write(x=''):
                           "submitty/autograding-default:latest",
                           "submitty/java:11",
                           "submitty/python:3.6",
-                          "submittyrpi/csci1200:default"
+                          "submittyrpi/csci1200:default",
+                          "rust:1.58-slim"
```

Comment:
Let's move this so that it's added in `.setup/install_system.sh` only on vagrant systems, and not have it be installed by default for all Submitty installations.